torch/distributions/lkj_cholesky.py

```diff
@@ -23,15 +23,17 @@ class LKJCholesky(Distribution):
     to make the probability of the correlation matrix :math:`M` generated from
     a Cholesky factor proportional to :math:`\det(M)^{\eta - 1}`. Because of that,
     when ``concentration == 1``, we have a uniform distribution over Cholesky
-    factors of correlation matrices. Note that this distribution samples the
+    factors of correlation matrices.::
```

Comment:
Remove full stop before colon?

Reply:
Sure, makes sense. I though the double colon was just to produce the next box, but as it is rendered, of course, it needs to be removed.

```suggestion
    factors of correlation matrices::
```